python/examples/simple_pipeline.py

```diff
@@ -7,75 +7,65 @@
 import erdos
 import time
 
+from erdos.operator import Source, Sink
 
-class SendOp(erdos.Operator):
-    def __init__(self, write_stream):
-        self.write_stream = write_stream
 
-    @staticmethod
-    def connect():
-        return [erdos.WriteStream()]
+class SendOp(Source):
+    def __init__(self):
+        print("initializing source op")
 
-    def run(self):
+    def run(self, write_stream):
         count = 0
         while True:
             msg = erdos.Message(erdos.Timestamp(coordinates=[count]), count)
             print("SendOp: sending {msg}".format(msg=msg))
-            self.write_stream.send(msg)
+            write_stream.send(msg)
 
             count += 1
             time.sleep(1)
 
 
-class CallbackOp(erdos.Operator):
-    def __init__(self, read_stream):
-        print("initializing  op")
-        read_stream.add_callback(CallbackOp.callback)
+class CallbackOp(Sink):
+    def __init__(self):
+        print("initializing callback op")
 
-    @staticmethod
-    def callback(msg):
-        print("CallbackOp: received {msg}".format(msg=msg))
+    def run(self, read_stream):
+        print("Executing run with the ReadStream {}".format(read_stream))
```

Comment:
Can you please remove this? This example is supposed to show only the callback-driven execution. `PullOp` and `TryPullOp` exemplify a `run`-driven execution.